components/vocabularies/api/src/main/java/org/phenotips/vocabulary/internal/solr/AbstractSolrVocabulary.java

```diff
@@ -241,6 +245,18 @@ public long getDistance(VocabularyTerm fromTerm, VocabularyTerm toTerm)
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public String getSourceLocation()
+    {
+        return this.relocationService.getRelocation(this.getDefaultSourceLocation());
+    }
+
+    @Override
+    public List<VocabularyExtension> getExtensions()
+    {
+        return this.extensions.get();
```

Comment:
Does this return all extensions, or just those valid for this vocabulary? It should be the latter.